specification/compiler/model/metamodel.ts

```diff
@@ -251,8 +277,13 @@ export class TypeAlias extends BaseType {
 export enum Stability {
   stable = 'stable',
   beta = 'beta',
-  experimental = 'experimental'
-  // NOTE: do we filter out "private"?
+  experimental = 'experimental',
+  TODO = 'TODO'
+}
+export enum Visibility {
+  public = 'public',
+  beta = 'feature_flag',
+  experimental = 'private'
```

Comment:
Failed search/replace? Should the identifiers be `feature_flag` and `private`?

Reply:
yep, updated!